fontforge/splinefill.c

```diff
@@ -1787,7 +1787,7 @@ BDFFont *SplineFontPieceMeal(SplineFont *sf,int layer,int ptsize,int dpi,
 	bdf->recontext_freetype = bdf->unhinted_freetype = false;
     }
     
-    if ( (ftc || bdf->recontext_freetype || bdf->recontext_freetype) && (flags&pf_antialias) )
+    if ( (ftc || bdf->recontext_freetype || bdf->unhinted_freetype) && (flags&pf_antialias) )
```

Comment:
@serval2412, how did you discover/verify that bdf->unhinted_freetype is to be checked here?